packages/pyright-internal/src/backgroundThreadBase.ts

```diff
@@ -62,6 +60,12 @@ export class BackgroundThreadBase {
         if (!this._serviceProvider.tryGet(ServiceKeys.tempFile)) {
             this._serviceProvider.add(ServiceKeys.tempFile, new RealTempFile());
         }
+
+        // Stash the base directory into a global variable.
```

Comment:
It was previously, just this happened a couple of lines above. Line 51 in the old code. I can't do it now until the file system and tmp service are there, otherwise we can't figure out the case sensitivity.